internal/engine/eval/vulncheck/report.go

```diff
@@ -0,0 +1,331 @@
+		return "", fmt.Errorf("could not parse dependency template: %w", err)
+	}
+	rowsTmpl, err := htmltemplate.New(tableVulnerabilitiesRowsName).Parse(tableVulnerabilitiesRows)
+	if err != nil {
+		return "", fmt.Errorf("could not parse vulnerability template: %w", err)
+	}
+
+	var summary strings.Builder
+	if len(r.TrackedDependencies) == 0 {
+		summary.WriteString(noVulsFoundText)
+		return summary.String(), nil
+	}
+
+	var headerBuf bytes.Buffer
+	if err := headerTmpl.Execute(&headerBuf, nil); err != nil {
+		return "", fmt.Errorf("could not execute template: %w", err)
+	}
+	summary.WriteString(headerBuf.String())
+
+	for i := range r.TrackedDependencies {
+		var rowBuf bytes.Buffer
+
+		if err := rowsTmpl.Execute(&rowBuf, struct {
+			DependencyEcosystem string
+			DependencyName      string
+			DependencyVersion   string
+			Vulnerabilities     []Vulnerability
+		}{
+			DependencyEcosystem: r.TrackedDependencies[i].Dependency.Ecosystem.AsString(),
+			DependencyName:      r.TrackedDependencies[i].Dependency.Name,
+			DependencyVersion:   r.TrackedDependencies[i].Dependency.Version,
+			Vulnerabilities:     r.TrackedDependencies[i].Vulnerabilities,
+		}); err != nil {
+			return "", fmt.Errorf("could not execute template: %w", err)
+		}
+		summary.WriteString(rowBuf.String())
+	}
+	summary.WriteString(tableVulnerabilitiesFooter)
+
+	reviewBody, err := render(minderTemplateName, minderTemplateString, minderTemplateData{
+		MagicComment: reviewBodyMagicComment,
+		Body:         summary.String(),
+		Footer:       contactString,
+	})
+	if err != nil {
+		return "", fmt.Errorf("could not create review body: %w", err)
+	}
+
+	return reviewBody, nil
+}
+
+const (
+	bugHtmlEmoji     = "&#128030;"
+	fixHtmlEmoji     = "&#128736;"
+	warningHtmlEmoji = "&#9888;&#65039;"
+	successHtmlEmoji = "&#9989;"
+	reviewHtmlEmoji  = "&#128202;"
+)
+
+const (
+	reportTemplateName   = "reportBody"
+	reportTemplateString = `
+{{- $vulnerabilityCount := .Metadata.VulnerabilityCount -}}
+{{- $remediationCount := .Metadata.RemediationCount -}}
+{{- $successSymbol := .Symbols.success -}}
+{{- $warningSymbol := .Symbols.warning -}}
+{{- $countVulnsWithFix := .CountFixedVulnerabilities -}}
+<h2>Minder Vulnerability Report {{ if gt $vulnerabilityCount 0 }}{{$warningSymbol}}{{ else }}{{$successSymbol}}{{ end }}</h2>
+<p>{{ .Report.StatusText }}</p>
+<blockquote>
+<h3>Vulnerability scan of <code>{{slice .Report.CommitSHA 0 8 }}:</code></h3>
+{{- if .Report.ReviewUrl }}<a href=" {{ .Report.ReviewUrl }}">{{ .Symbols.review }} <b>View Full Review</b></a>{{ end -}}
+<ul>
+	<li>{{ .Symbols.bug }} <b>vulnerabilities:</b> <code>{{ $vulnerabilityCount }}</code></li>
+	<li>{{ .Symbols.fix }} <b>fixes:</b> <code>{{ $remediationCount }}</code></li>
+</ul>
+</blockquote>
+{{- if .Report.TrackedDependencies }}
+<table>
+	<thead>
+		<tr>
+			<th>Package</th>
+			<th>Version</th>
+			<th>#Vulnerabilities</th>
+			<th>#Fixes</th>
+			<th>Patch</th>
+	  	</tr>
+	</thead>
+	<tbody>
```

Comment:
For reason I really don't understand the `<tbody>` tag is not rendered correctly for me, see [this example](https://github.com/jakubtestorg/bad-python/pull/79#issuecomment-1923752884).

I wonder if we need the thead and tbody tags at all, can't we get away with just tr?

Reply:
As far as I understand, using the `Issues` API is not actually an issue since every PR is technically an issue (according to the GitHub docs). 

See https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#about-issue-and-pull-request-comments:
> You can use the REST API to create and manage comments on issues and pull requests. Every pull request is an issue, but not every issue is a pull request. For this reason, "shared" actions for both features, like managing assignees, labels, and milestones, are provided within the Issues endpoints. To manage pull request review comments, see "[Pull request review comments](https://docs.github.com/en/rest/pulls/comments)."